.travis.yml

```diff
@@ -1,39 +0,0 @@
-language: android
```

Comment:
We'll also need to remove the travis config from GitHub but I'm not sure if you have access. I can take care of this after this lands.